packages/core/src/common/item.store.ts

```diff
@@ -3,14 +3,11 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
+import type { ItemObject } from "@k8slens/list-layout";
 import autoBind from "auto-bind";
 import orderBy from "lodash/orderBy";
 import { action, computed, observable, when, makeObservable } from "mobx";
 
-export interface ItemObject {
```

Comment:
Does this cause issues with the extension api?

Reply:
As this is not imported anywhere, I don't see how it could cause issues in extension api. If something that is importing this from `@k8slens/list-layout` would cause it to end up in extension api, I still don't see an issue. 🤔